scripts/patch-rpath

```diff
@@ -11,7 +11,7 @@ echo "Using final root directory of $root"
 
 cd $root
 
-for f in $(find bin lib jre -type f -executable -or -name "*.so*"); do
+for f in $(find bin lib jre -type f -executable -or -name "*.so*" | grep -v libc.so.6 | grep -v libpthread.so.0); do
```

Comment:
Can I ask why we need to ignore these to shared objects? 
Furthermore, can they be turned into one grep statement that has a comment or such to help future developers understand why we care about these?

Reply:
These are skipped because `patchelf` mangles them. Luckily they do not have dependencies that need to be loaded, so they don't actually need to get patched.